google/ggrc-core
# GGRC-5124 Remove legacy roles

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [x] #7826
- [x] #7824 
- [x] #7822 
- [x] #7821 
- [x] #7820 
- [x] #7819 
- [x] #7818 
- [x] #7817 

# Issue description

Now that all user roles have been moved to ACL we must purge the system of the legacy roles and role entries.

The only roles that must remain in the system are global roles.

# Steps to test the changes

Do a basic rbac check for programs, workflows and audits.
Also verify that there are no more references to old roles in our code.

# Solution description

Removal of all code that handles obsolete roles.

# Sanity checklist


# Migration checklist

# PR Review checklist
